.github/workflows/weekly-report.yml

```
@@ -17,42 +17,68 @@ jobs:
-            ].join('\n');
+            ].filter(Boolean).join('\n');
 
             core.setOutput('msg', text);
+
       - name: Send Weekly Report to Discord
         uses: Ilshidur/action-discord@master
         env:
           DISCORD_WEBHOOK: ${{ secrets.DISCORD_WEBHOOK }}
         with:
-          args: ${{ steps.build.outputs.msg }}
+          args: ${{ steps.build.outputs.msg }
```

actionlint (1.7.7)

got unexpected EOF while lexing end marker }}, expecting '}'

(expression)